README.md

```diff
@@ -24,3 +24,7 @@ You can add the pattern library package as a dependency to your JavaScript appli
 ```
 npm install @cloudfour/patterns
 ```
+
+## Contributing
+
+Please see the [contribution guidelines for this project](CONTRIBUTING.md).
```

Comment:
Is it possible to make this a tad more specific?

To learn how to run this project locally, create new things and modify existing things...

Reply:
good call — updated